frodo.js

```diff
@@ -2,21 +2,21 @@
  * Extras Plugin
```

Comment:
This function might benefit from using async/await instead of callbacks for better readability.